src/pybind/mgr/dashboard/run-tox.sh

```diff
@@ -17,11 +17,14 @@ fi
 
 source ${MGR_DASHBOARD_VIRTUALENV}/bin/activate
 
+ENV_LIST="cov-init,"
+if [ "$WITH_PYTHON2" = "ON" ]; then
```

Comment:
Shouldn't we declared a default value for `WITH_PYTHON2` in the beginning of the script (as we do with `WITH_PYTHON3`) so that the script runs well outside ctest.